arch/arm64/configs/bcmrpi3_defconfig

```diff
@@ -1216,6 +1216,7 @@ CONFIG_MAXIM_THERMOCOUPLE=m
 CONFIG_MAX31856=m
 CONFIG_PWM_BCM2835=m
 CONFIG_PWM_PCA9685=m
+CONFIG_BRCMSTB_L2_IRQ=y
```

Comment:
Why enable it for Pi3 defconfig? That seems odd?

Reply:
Good point - I'll remove it.

Reply:
(not entirely sure of the use of bcmrpi_3_defconfig - we use bcm2711_defconfig for official 64-bits on Pi3 and Pi4).

Reply:
We could get rid of it, except that people are using it and might want the slightly smaller kernel on Pi 3.